osbrain/tests/test_agent.py

```diff
@@ -50,7 +50,9 @@ def test_early_agent_proxy(nsaddr):
     # Locate agent now
     a0 = Proxy('a0', timeout=3.)
     # Just check agent is ready
-    assert a0.ping() == 'pong'
+    safe_status = a0._safe
+    assert a0.unsafe.ping() == 'pong'
+    a0._safe = safe_status
```

Comment:
Oh, I see. So I will remove the status storage from the code ^^. I think we can merge this into master after #74